src/Data/Scripts/Script_001_CreateTables.sql

```diff
@@ -0,0 +1,48 @@
+CREATE TABLE
+    IF NOT EXISTS "User" (
```

Comment:
![image](https://github.com/user-attachments/assets/7bf9c067-d5f3-44f4-981f-ff9a45d40ca4)

This way to create tables, is creating tables only User with Uppercase first letter and other Lower

Reply:
I changed the values to preserve the consistency